pa_config/rca.conf

```diff
@@ -58,6 +58,11 @@
     "cache-config": {
       "field-data-cache-size-threshold" : 0.8,
       "shard-request-cache-threshold" : 0.9
+    },
+    // Needs to be updated as per the performance test results
+    "cache-decider-config": {
```

Comment:
Should `decider` config really be nested inside `rca config` ?

Reply:
Created decider-config-settings